ydl_podcast/__init__.py

```diff
@@ -193,25 +200,37 @@ def cleanup(sub):
 
 def write_xml(sub):
     directory = os.path.join(sub['output_dir'], sub['name'])
+
+    image_tag = ""
+
+    if sub['image'] is not None:
```

Comment:
this will `KeyError` if the image key doesn't work, can you switch to `sub.get('image')` ?